trulens_eval/examples/expositional/models/bedrock_finetuning_experiments.ipynb

```diff
@@ -55,7 +55,7 @@
    },
```

Comment:
<p>Is this change intentional?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/795/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yes, with the latest SM SDK. It expects the name as above </p>